### ISSUE TITLE
CAPE sandbox analyzer

### ISSUE THREAD

**Issue by mlodic** [intelowlproject/IntelOwl#722]
https://www.capesandbox.com

**Comment by 0x0elliot** [comment]
Hey! Can I take up this issue? I am free now and I wanted to contribute back again.

**Comment by mlodic** [comment]
We are not sure that they still activate new accounts...

**Comment by 0x0elliot** [comment]
The only option we have is to wait.

**Comment by kevoreilly** [comment]
We do activate new accounts when a manual request is made via Twiter - we cannot automate the activation as we have suffered so much abuse from anonymous/bogus accounts who only wish to scrape. This has caused denial of service for all users, I can only apologise for this but it is a necessary evil.

**Comment by 0x0elliot** [comment]
I will try making a manual request on Twitter then. Thanks kev!

**Comment by 0x0elliot** [comment]
I haven't been able to figure out where the API authentication token for https://www.capesandbox.com/ is? I looked through the documentation and couldn't find anything myself. I have gotten my account verified for context. Anyone else try their hand here and see if they can help?

**Comment by mlodic** [comment]
Hi @kevoreilly , really appreciated! I'll ask now on Twitter too so I can help with the integration!

**Comment by 0x0elliot** [comment]
Matteo, you can use my account too! I don't plan on reusing it anyway.

**Comment by mlodic** [comment]
https://capev2.readthedocs.io/en/latest/usage/api.html?highlight=api

You just need to call `curl -d "username=<USER>&password=<PASSWD>" https://capesandbox.com/apiv2/api-token-auth/` with your credentials and you'll get the token to use. 

Then the documentation and the examples of the API are [here](https://capesandbox.com/apiv2/)

**Comment by 0x0elliot** [comment]
Thanks!

**Comment by 0x0elliot** [comment]
@kevoreilly hey Kevin!

I noticed that whenever I was uploading a file using the API, it seemed stuck at `pending` while when I uploaded it through the GUI at https://www.capesandbox.com/analysis it moved from `pending` to `running` quicker.

Is there something regarding this that you're aware of? I have pretty much finished this one.

**Comment by kevoreilly** [comment]
Hmm the only thing I can think of is that there have recently been issues (now should be fixed) that caused jobs to get stuck in pending - although it shouldn't be different for api to web ui... Is this something you have noticed recently on capesandbox.com?

**Comment by 0x0elliot** [comment]
If I remember correctly, Then yes it does seem like an issue I have noticed recently. My test image of the md5 hash `98d435933b2322d4c8e8b09c0be9080b` which was uploaded through the APIs has been stuck since `Feb. 3, 2022, 3:33 p.m.` (Official mentioned time. It should be 7 PMish now for reference for whichever timezone it is using as a reference.)

**Comment by 0x0elliot** [comment]
Yup seems like the "pending" issue has been taken care of. Fewer files are pending and the queue is moving smoothly. Thanks Kev!
